### DISCUSSION TITLE
How to withdraw the email verification of global campus of a locked account?

### DISCUSSION THREAD

**Discussion by quaerent** [community/community#70025] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm an undergraduate student who want to sign up for the global campus and use Github Copilot.

Actually I've already done that in another account. But I recently lost my computer and now I cannot pass the 2FA security verification, which means I lose my access to the account.

So I unlinked the email I used to link with the old account, and created a new one. However, since the student email is already used to sign up for the campus for my old account, I cannot use it to sign up for my new account.

Is there any way that I can solve this? This is really important and urgent for me, THANK YOU!!!

**Comment by quaerent** [comment]
The best way I can figure out is to contact github staff and try to get my access to the old account back, but I have absolutely no idea how.

**Comment by wellcezarioo** [comment]
I'm having the same problem, were you able to solve it with support?
